### PR TITLE
Added security help section and cve table 

### PR DESCRIPTION
**Pull request by atharv-g-kulkarni**

### CODE REVIEW

File: src/components/blog/CveTable.vue [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  ]
+</script>
+
+<style scoped>
+  .cve-table-mainContainer {
+    padding: 32px 0px;
+    position: relative;
+    left: 50%;
+    margin-left: -50vw;
+    width: 100vw;
```
Comment (rohit-bharmal) [inline src/components/blog/CveTable.vue]: Don't use vw. add, give as 100%

File: src/components/blog/SecurityHelpSection.vue [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+          Receive email notifications of security updates, bug fixes, and
+          enhancements, also known as errata.
+        </p>
+        <p>
+          Errata notifications are controlled based on your method of
+          subscription management.
+        </p>
+        <rh-cta slot="footer">
+          <a
+            href="https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/auth?response_type=code&client_id=https%3A%2F%2Fwww.redhat.com%2Fwapps%2Fugc-oidc&redirect_uri=https%3A%2F%2Fwww.redhat.com%2Fwapps%2Fugc%2Fsso%2Flogin&state=ac355c1d-09be-4834-aa49-d7a6135a0143&login=true&scope=openid"
```
Comment (rohit-bharmal) [inline src/components/blog/SecurityHelpSection.vue]: change this link. Its SSO link of RedHat

Comment (hybridx) [reply src/components/blog/SecurityHelpSection.vue]: Please do not add internal links

File: src/components/blog/CveTable.vue (hunk 1 above)
Comment (atharv-g-kulkarni) [reply src/components/blog/CveTable.vue]: removed vw

File: src/components/blog/SecurityHelpSection.vue (hunk 2 above)
Comment (atharv-g-kulkarni) [reply src/components/blog/SecurityHelpSection.vue]: changed the links

**Review by hybridx — APPROVED** [review]

**Pull request merged**